plugins/scaffolder/src/components/Router.tsx

```diff
@@ -77,6 +77,10 @@ export type RouterProps = {
     /** Whether to show a link to the actions documentation */
     actions?: boolean;
   };
+  /**
+   * Options for any filter on the scaffolder page
+   */
+  customFilters?: Array<React.ReactElement>;
```

Comment:
This doesn't use the same pattern for adding filters as existing usage in plugins. @benjdlambert how does this relate to scaffolder next? are custom filters for the template list page handled as part of that?